packages/material-ui/src/utils/muiComponent.js

```diff
@@ -0,0 +1,14 @@
+export default function muiComponent(useStyles, Component) {
```

Comment:
I guess it makes more sense to use the component as the first parameter. `useStyles` is more likely to get removed at some point than `Component`

Reply:
I can't. If I swap the arguments, react-docgen does no longer "see" the component in the file.

Reply:
This should be addressed in reactjs/react-docgen#343